### PR TITLE
Add a bunch more stuff to shuffleboard

### PR DESCRIPTION
**Pull request by balghane**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -271,6 +271,33 @@ public void initSendable(SendableBuilder builder) {
...
+        "Intake Deploy Velocity (deg/s)", deployMotorEncoder::getVelocity, null);
     builder.addBooleanProperty("See Game Piece", this::seeGamePiece, null);
+    builder.addBooleanProperty(
+        "Desire clamped",
+        () -> {
+          return desireClamped;
+        },
+        null);
+    builder.addBooleanProperty("At Desired Pos", this::atDesiredPosition, null);
+    builder.addDoubleProperty("Intake wheel power", intakeLeft::get, null);
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/Intake.java]: Units?

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 2]
```diff
@@ -439,10 +439,42 @@ public void initSendable(SendableBuilder builder) {
...
     SendableHelper.addChild(builder, this, elevatorController, "ElevatorController");
     builder.addDoubleProperty(
-        "Elevator Position", elevatorLeftEncoder::getPosition, elevatorLeftEncoder::setPosition);
-    builder.addDoubleProperty("Arm Position", armEncoder::getPosition, armEncoder::setPosition);
-    builder.addBooleanProperty("Done Scoring", this::doneScoring, null);
+        "Elevator Position (in)",
+        elevatorLeftEncoder::getPosition,
+        elevatorLeftEncoder::setPosition);
+    builder.addDoubleProperty(
+        "Elevator Vel (in)", elevatorLeftEncoder::getVelocity, elevatorLeftEncoder::setPosition);
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/Lift.java]: Why is the setter `setPosition`? I don't think there should be a setter here.

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 3]
```diff
@@ -439,10 +439,42 @@ public void initSendable(SendableBuilder builder) {
...
+    builder.addDoubleProperty(
+        "Elevator Left Abs Pos (deg)", elevatorLeftAbsEncoder::getPosition, null);
+    builder.addDoubleProperty(
+        "Elevator Right Abs Pos (deg)", elevatorRightAbsEncoder::getPosition, null);
+    builder.addBooleanProperty("Clear of intake", this::clearOfIntakeZone, null);
+    builder.addDoubleProperty(
+        "Arm Abs Position (deg)", armAbsoluteEncoder::getPosition, armEncoder::setPosition);
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/Lift.java:452]: I ~~know~~ edit: think we can't setPosition with the abs encoder, so I think it makes more sense to just not have a setter. It feels a little misleading.

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 4]
```diff
@@ -439,10 +439,42 @@ public void initSendable(SendableBuilder builder) {
...
+    builder.addDoubleProperty(
+        "Arm Position (deg)", armEncoder::getPosition, armEncoder::setPosition);
+    builder.addDoubleProperty("Arm Vel (deg/s)", armEncoder::getVelocity, armEncoder::setPosition);
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/Lift.java]: Same as elevator velocity comment on line 446.

**Review by tails618 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Lift.java (hunk 3 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/Lift.java:452]: I think we can, it's just not really advisable.. but it's hard to accidentally set things anyway (didn't we figure out it only works in test mode?)

File: src/main/java/frc/robot/subsystems/Lift.java (hunk 4 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/Lift.java]: Whoops copy paste error removed

File: src/main/java/frc/robot/subsystems/Lift.java (hunk 2 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/Lift.java]: Whoops copy paste error removed

File: src/main/java/frc/robot/subsystems/Intake.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/Intake.java]: It's just -1 to 1, added

File: src/main/java/frc/robot/subsystems/Lift.java (hunk 3 above)
Comment (tails618) [reply src/main/java/frc/robot/subsystems/Lift.java:452]: Yup, that's true. In that case I think it's fine.

**Review by tails618 — APPROVED** [review]

**Pull request merged**